tests/wpt/mozilla/tests/mozilla/bluetooth/connect/get-same-gatt-server.html

```diff
@@ -6,16 +6,10 @@
 'use strict';
 promise_test(() => {
     window.testRunner.setBluetoothMockDataSet(adapter_type.heart_rate);
-    return window.navigator.bluetooth.requestDevice(
-        { filters: [{services: [heart_rate.name]}]})
-    .then(device => {
-        return Promise.all([device.gatt.connect(), device.gatt.connect()]);
-    })
-    .then(gattServers => {
-        return assert_true(gattServers[0], gattServers[1]);
-    })
-    .catch(error => {
-        assert_unreached('Promise should have resolved. ' + error);
-    });
```

Comment:
Personally, I think it's more readable when the return value is on its own line.